module/dice/damage-roll.mjs

```diff
@@ -173,7 +175,13 @@ export default class DamageRoll extends Roll {
 
           // Powerful critical - maximize damage and reduce the multiplier by 1
           if ( this.options.powerfulCritical ) {
-            let bonus = term.number * term.faces;
+            const bonus = BasicRoll.create({
+              parts: [term.formula],
+              data: this.data,
+              situational: false,
+              options: this.options
+            }).evaluateSync({ maximize: true })
+              .total ?? 0;
```

Comment:
Since all we have is a simple `DiceTerm` here, I don't think we need to pass all these extra options to `BasicRoll`. Note that `Roll.create` will automatically redirect to `BasicRoll`, so we do not need to explicitly import it here.

```suggestion
            const bonus = Roll.create(term.formula).evaluateSync({ maximize: true }).total ?? 0;
```